README.md

```diff
@@ -56,6 +55,13 @@ For more info on ECS, check the [ECS FAQ](https://github.com/SanderMertens/ecs-f
 - [Query Manual](docs/Queries.md) ([docsforge](https://flecs.docsforge.com/master/query-manual/))
 - [Relations Manual](docs/Relations.md) ([docsforge](https://flecs.docsforge.com/master/relations-manual/))
 
+## How to use and build
+The easiest way to add Flecs to a project is to add [flecs.c](https://raw.githubusercontent.com/SanderMertens/flecs/master/flecs.c) and [flecs.h](https://raw.githubusercontent.com/SanderMertens/flecs/master/flecs.h) to your source code. These files can be added to both C and C++ projects (the C++ API is embedded in `flecs.h`).
+
+Alternatively you can also build Flecs as a standalone library by using the [bake](https://github.com/SanderMertens/bake) (cmake, meson, bazel are also supported).
```

Comment:
Would change this to:

> Flecs can also be built as a standalone library by using cmake, meson, bazel or [bake](https://github.com/SanderMertens/bake).

I don't want to over-emphasize bake since most people will likely not use it, and I don't want to create the impression that you need bake to use flecs.

Reply:
I thought a bit of self-promotion wouldn't hurt here, ~~and you can be free of guilt as well by not putting it here yourself xP~~, but yeah, no problem.